packages/packherd-require/src/sourcemap-support.ts

```diff
@@ -213,7 +213,7 @@ class SourcemapSupport {
       frame = cloneCallSite(frame)
 
       frame.getFileName = function getFileName () {
-        return pos.source || pos.name || null
+        return pos.source || pos.name || undefined
```

Comment:
This type check was failing, likely due to upgrading types/node.

![Screenshot 2024-02-20 at 4 23 57 PM](https://github.com/cypress-io/cypress/assets/1271364/78c66f23-1768-4ec1-acf2-49e81f7da71c)